src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import Card from "./components/Card"
+import data from "./data"
+
+const App = () => {
+
+  const handleClick = () => window.alert("Clicked!")
```

Comment:
Why is this here? I think there no where to use this function, if so... you should remove it!